src/main/scala/org/globalforestwatch/layers/GADMadm0.scala

```diff
@@ -3,16 +3,264 @@ package org.globalforestwatch.layers
 import org.globalforestwatch.grids.GridTile
 
 case class GADMadm0(gridTile: GridTile, kwargs: Map[String, Any]) 
```

Comment:
It's not part of this PR per se, but for some reason I feel something is off about the casing... should ie be like GADMAdm0? GadmAdm0? It's hard with acronyms.